### PR TITLE
Add Clerky

### PR DESCRIPTION
**Pull request by akshayrawat**
clerky_logo_hexbin

like [this](https://github.com/clerky/hexbin/blob/db91fde6e45c159b722b9b3c36d02d1d9a38050d/hexagons/clerky.png)

### CODE REVIEW

**Comment by derhuerst** [comment]
Is there an SVG version of this really simple logo? (:

**Comment by akshayrawat** [comment]
@derhuerst Its already in this PR. Did I miss something?

**Comment by derhuerst** [comment]
Sorry, I missed something. :flushed:

File: meta/clerky.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "clerky",
+  "author": "Clerky, Inc.",
+  "license": "None",
+  "vector": "http://hexb.in/vector/clerky.svg",
+  "description": "https://www.clerky.com",
```
Comment (Ruxton) [inline meta/clerky.json:6]: Please provide a more useful description

File: meta/clerky.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "clerky",
+  "author": "Clerky, Inc.",
+  "license": "None",
```
Comment (Ruxton) [inline meta/clerky.json:4]: Please provide an accurate license

**Review by Ruxton — CHANGES_REQUESTED** [review]

**Pull request closed without merging**